metaproject.py

```diff
@@ -66,6 +66,39 @@ def debug_log_message(message, tag, level):
 
     QgsApplication.exitQgis()
 
+def yaml2qgs(argv):
+    parser = argparse.ArgumentParser('Generate QGIS project from a YAML file.')
```

Comment:
Would be nice to have one single entry point with one parser that calls other methods based on arguments.